python/hail/methods/statgen.py

```diff
@@ -2144,10 +2193,9 @@ def genetic_relatedness_matrix(dataset) -> KinshipMatrix:
 
     return KinshipMatrix._from_block_matrix(tstr,
                                             grm,
-                                            [row.s for row in dataset.cols().select('s').collect()],
```

Comment:
This should really be the whole col_key, right?

Reply:
yes for this method, `genetic_relatedness_matrix`.

Reply:
I think we all agree it should get fixed, but not in this PR.

Reply:
Yes for GRM, but once this PR goes in, you can run GRM so long as the column key is a single string. RRM already has this change.